src/transformers/configuration_pegasus.py

```diff
@@ -0,0 +1,62 @@
+""" PEGASUS model configuration """
+
+import logging
+
+from .configuration_bart import BartConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+class PegasusConfig(BartConfig):
+    r"""
+        :class:`~transformers.PegasusConfig` is the configuration class to store the configuration of a
+        `PegasusModel`.
+    """
+    model_type = "pegasus"
+
```

Comment:
It would be great to add docstrings here for arguments.

Reply:
+!

Reply:
-2